purchases/src/main/kotlin/com/revenuecat/purchases/common/FileHelper.kt

```diff
@@ -34,21 +33,21 @@ internal class FileHelper(
         return file.delete()
     }
 
-    // This is using a lambda with a Stream instead of returning the Stream itself. This is so we keep
-    // the responsibility of closing the bufferedReader to this class. Note that the Stream should
-    // be used synchronously, otherwise the bufferedReader will be closed before the stream is used.
+    // This is using a lambda with a Sequence instead of returning the Sequence itself. This is so we keep
+    // the responsibility of closing the bufferedReader to this class. Note that the Sequence should
+    // be used synchronously, otherwise the bufferedReader will be closed before the Sequence is used.
     @RequiresApi(Build.VERSION_CODES.N)
```

Comment:
I believe this is probably not needed anymore, as you mentioned over slack.

Reply:
Alright, will create a follow-up PR to clean these up if we decide they're not needed.

Reply:
For reference: https://github.com/RevenueCat/purchases-android/pull/1944